sh_tools.py

```diff
@@ -261,12 +266,18 @@ def copy_remote_file(input_remote_file, output_local_file, inputDBS='global', n_
     os.remove(output_local_file)
 
   if len(pfns_list) == 0:
-    raise RuntimeError(f'Unable to find any remote location for {input_remote_file}.')
+    if input_remote_file.startswith('/store/'):
+      if inputDBS=='phys03': # Probably a Tau-embedded sample stored on root://cmsxrootd-kit.gridka.de/
+        pfns_list = ["root://cmsxrootd-kit.gridka.de/"+input_remote_file, input_remote_file]
```

Comment:
I suggest a universal solution: add `custom_pfns_prefix=None` as an optional argument to `copy_remote_file` (and to nanoProdWrapper etc.). In that case, it will work for any privately produced samples.